### PR TITLE
Added basic FAQ and contact us pages

### PR DESCRIPTION
**Pull request by dhruvapte26**
In reference to Issue #5  :
- An FAQ page has been created with some basic questions
- a small 'contact us' page that includes our Facebook and Meetup page links 

PR open to constructive criticism and reviews :smile:

### CODE REVIEW

File: Contact-us.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: base
+---
+<div class="row">
+  <div class="col-md-12">
+    <h3>Contact Us</h3>
+    <br><br>
+    
+    <p> Contacting the Google Developers Group Vadodara is not a Herculean task. You just need to be in the right places:</p>
```
Comment (manan-jadhav) [inline Contact-us.html]: Not "the"; Baroda, not vadodara

File: _posts/2018-07-15-GDG-WTM-Meetup-on-Agile.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+categories: ["Meetups"]
+tags: ["Agile", "Software Development"]
+author: Dhruv Apte
+assets: "/assets/2018-07/GDG-WTM-Meetup-on-Agile"
+---
+
+This was the first Women Tech-makers Meetup in the year 2018. The topic was AGILE- by Kratika 
+Shah.
+
+WTM is the sister branch of GDG to promote women in software. In Vadodara, the creation of a [WTM Vadodara](https://twitter.com/WTMBaroda) chapter was announced in October 2017
```
Comment (manan-jadhav) [inline _posts/2018-07-15-GDG-WTM-Meetup-on-Agile.md]: Baroda, not Vadodara

**Review by manan-jadhav — CHANGES_REQUESTED** [review]
Please make changes as noted

**Comment by dhruvapte26** [comment]
@CurosMJ requested changes have been made.

**Review by manan-jadhav — APPROVED** [review]

**Pull request merged**